go/exoservice/index.go

```diff
@@ -0,0 +1,27 @@
+package exoservice
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/exosocket"
+)
+
+// Bootstrap is the high level API to Exocom
+//   create an ExoRelay instance using environment variables
+//   connects to ExoCom
+//   registers the given handlers
+func Bootstrap(messageHandlers exorelay.MessageHandlerMapping) {
+	config := exosocket.NewEnvConfig()
+	exoRelay := exorelay.NewExoRelay(config)
+	err := exoRelay.Connect()
+	if err != nil {
+		panic(fmt.Sprintf("Error connecting: %v", err))
```

Comment:
Do we want Exocom to panic if there is an error? Why not return with an error here?

Reply:
I prefer this API as its very high level where the user doesn't want to have to care about calling a method that returns an error. This is only one step below a CLI.